app/src/features/Preview/Preview.tsx

```diff
@@ -166,8 +172,9 @@ const Preview = (): JSX.Element => {
     if (exploration && mappingId && mapping?.primary_key_column) {
       const primaryKey = mapping.primary_key_column;
       const primaryKeyIndex = exploration.fields.indexOf(primaryKey);
-      const primaryKeyValue = exploration.rows[index]?.[primaryKeyIndex];
-
+      const primaryKeyValue = exploration.rows[index]?.[
+        primaryKeyIndex
+      ]?.toString();
```

Comment:
what does this fix ? doesn't seem related to this PR

Reply:
I had a big error, i'm deleting it and make another pr

Reply:
I had a big error when the value was a number... I'm going to delete it and make another PR.